connectivity/FEATURE_BLE/CMakeLists.txt

```diff
@@ -1,11 +1,22 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+add_library(mbed-ble OBJECT)
+# cordio libraries are part of ble stack (_ll is optional)
+# We use INTERFACE libraries to form the main object library otherwise we would 
+# get circular deps error between ble <-> cordio
+add_library(mbed-ble-cordio INTERFACE)
+add_library(mbed-ble-cordio_ll INTERFACE)
+
+# Add drivers for BLE
+# Once we remove the old tools and their path requirements, we should move 
+# drivers to BLE stack to be in the same place
+add_subdirectory(../drivers/ble ${CMAKE_CURRENT_BINARY_DIR}/drivers/ble)
```

Comment:
Not certain if this is done properly. If I should rather just use `${MBED_PATH}/connectivity/drivers/ble` ? 

These are drivers for ble (extensions) so they should be included here rather than in driver.

Reply:
reverting this back to as it was (if we ever move drivers).